### ISSUE TITLE
Improve error message when project names contain spaces

### ISSUE THREAD

**Issue by TimothyJones** [pact-foundation/pact-js-core#249]
Similarly to #248, when the path contains a space, the ruby binaries don't work (#247). The error message is consistent and unhelpful.

We could detect this error message and recommend a fix / link to the docs.

A generic framework for detecting common errors and recommending fixes would be useful.

**Comment by mefellows** [comment]
I think this is resolved now by removing Ruby.

**Comment by TimothyJones** [comment]
Is Ruby removed? I thought it was still used for the CLI?

**Comment by mefellows** [comment]
Yes we still expose the binstubs, but I don't think we should change the behaviour of those.

I left another issue open that leaves this possibility open, however. I'm going to remove the remaining non FFI (i.e. Ruby) wrapper code in the next major version as we no longer have any major driver to continue to support those functions.

**Comment by TimothyJones** [comment]
I meant that this issue will affect the binstubs too.

I think it's really good to have the CLI wrapped in an npm package - it would be great to have that alive somehow even if they're removed from the main one.

**Comment by mefellows** [comment]
Ah yes, I suppose we can consider it alongside #248. I think at this moment bringing the binaries in the package is convenient (I'm not proposing we remove them), but doing anything more than that might be beyond what we should be doing here in this package.
